core/native/src/main/scala/it/unibo/scafi/PlatformDependentCostants.scala

```diff
@@ -0,0 +1,6 @@
+package it.unibo.scafi
```

Comment:
@KevinManciniHull can you solve this issue? It seems that there is a typo on the file name (Costants instead of Constants). Check also the other problem in the CI: https://www.codefactor.io/repository/github/scafi/scafi/pull/63